### PR TITLE
Fs jul19 konstantin b

### PR DESCRIPTION
**Pull request by ghost**
layout_human-rights solution by K Burnazaki

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <article>
```
Comment (gestap) [inline src/index.html]: use 2 whitespaces instead of 4

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+* {
```
Comment (gestap) [inline src/style.css]: `font` is an inherited style. No need to add it to `*`

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  margin-top: 0;
+  font-size: 17px;
+  font-weight: bold;
+}
+
+.sub-head {
+  font-weight: normal;
+  font-size: 15px;
+  margin-top: 1em;
+  margin-bottom: 1em;
```
Comment (gestap) [inline src/style.css]: try to write margin in one line

File: src/index.html (hunk 1 above)
Comment (burnazaki) [reply src/index.html]: done

File: src/style.css (hunk 2 above)
Comment (burnazaki) [reply src/style.css]: transferred to the body tag

File: src/style.css (hunk 3 above)
Comment (burnazaki) [reply src/style.css]: replaced w/ margin: 1em 0;

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
+* {
+  font-size: 13px;
```
Comment (gestap) [inline src/style.css]:
```suggestion
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+body {
+  font-family: Arial;
```
Comment (gestap) [inline src/style.css]:
```suggestion
  font-family: Arial;
  font-size: 13px;
```

**Review by ost-ptk — APPROVED** [review]

**Review by irenhh — APPROVED** [review]

**Pull request closed without merging**